app/src/main/java/org/wikipedia/talk/TalkPageSeenRepository.kt

```diff
@@ -0,0 +1,19 @@
+package org.wikipedia.talk
+
+import org.wikipedia.talk.db.TalkPageSeen
+import org.wikipedia.talk.db.TalkPageSeenDao
+
+class TalkPageSeenRepository constructor(private val talkPageSeenDao: TalkPageSeenDao) {
```

Comment:
What's the purpose of this class? It looks like a 1-to-1 wrapper around `TalkPageSeenDao`.

Reply:
Done.